server/parsers/basic_input_output_parser.ts

```diff
@@ -13,19 +13,79 @@ const sanitize = (content: string) => {
   return DOMPurify.sanitize(content, { FORBID_TAGS: ['img'] }).replace(/!+\[/g, '[');
 };
 
+const isEveryItemString = (array?: unknown) =>
+  !!(Array.isArray(array) && array.every((item) => typeof item === 'string'));
+
+export const parseSuggestedActions = (string: string): string[] => {
```

Comment:
Nit: maybe rename `string: string` -> `value: string`? as it sounds a bit odd that the variable name and type are the same.

Reply:
Sure, done.